### DISCUSSION TITLE
2021.06.07

### DISCUSSION THREAD

**Discussion by malaheaven** [malaheaven/issue-tracker#3] [General]
fe:
파일 첨부하기 기능 배제

**Comment by HongzCloud** [comment]
iOS: 오늘은 API 요청형식 작성했습니다~ 🔥

**Comment by 17-sss** [comment]
**FE**
- 오늘은 API 관련하여 회의를 하였고 반환되는 JSON 데이터에 대해 의논하였습니다.
- 이번 1주차에는 어떤 UI를 작업할지 회의하였습니다.
- FE 개발 환경을 설정 하였습니다.

**Comment by malaheaven** [comment]
##### BE
-  금일은 api 관련하여 회의를 진행했습니다. 
-  1주차 마일스톤 작성
- spring project init
